docs/setup_installation/admin/ha-dr/dr.md

```diff
@@ -2,174 +2,316 @@
+    provider: aws
+    config:
+        region: REGION
+        s3Url: ENDPOINT
+    credential:
+        key: cloud
+        name: hopsworks-bsl-credentials
+    objectStorage:
+        bucket: BUCKET_NAME
+        prefix: k8s_backup
+    EOF
+    ```
+
+- After the backup storage location becomes available, restore the backups. The following script restores the latest available backup. To restore a specific backup, set `backupName` instead of `scheduleName`.
+
+```bash
+echo "=== Waiting for Velero BackupStorageLocation  hopsworks-bsl to become Available ==="
+until [ "$(kubectl get backupstoragelocations hopsworks-bsl -n velero -o jsonpath='{.status.phase}' 2>/dev/null)" = "Available" ]; do
+  echo "Still waiting..."; sleep 5;
+done
+
+echo "=== Waiting for Velero to sync the backups from hopsworks-bsl ==="
+until [ "$(kubectl get backups -n velero -ojson | jq -r '[.items[] | select(.spec.storageLocation == "hopsworks-bsl")] | length' 2>/dev/null)" != "0" ]; do
+  echo "Still waiting..."; sleep 5;
+done
+
+
+# Restores the latest - if specific backup is needed then backupName instead
+echo "=== Creating Velero Restore object for k8s-backups-main ==="
+RESTORE_SUFFIX=$(date +%s)
+kubectl apply -f - <<EOF
+apiVersion: velero.io/v1
+kind: Restore
+metadata:
+  name: k8s-backups-main-restore-$RESTORE_SUFFIX
+  namespace: velero
+spec:
+  scheduleName: k8s-backups-main
+EOF
+
+echo "=== Waiting for Velero restore to finish ==="
+until [ "$(kubectl get restore k8s-backups-main-restore-$RESTORE_SUFFIX -n velero -o jsonpath='{.status.phase}' 2>/dev/null)" = "Completed" ]; do
+  echo "Still waiting..."; sleep 5;
+done
+
+# Restores the latest - if specific backup is needed then backupName instead
+echo "=== Creating Velero Restore object for k8s-backups-users-resources ==="
+kubectl apply -f - <<EOF
+apiVersion: velero.io/v1
+kind: Restore
+metadata:
+  name: k8s-backups-users-resources-restore-$RESTORE_SUFFIX
+  namespace: velero
+spec:
+  scheduleName: k8s-backups-users-resources
+EOF
+
+echo "=== Waiting for Velero restore to finish ==="
+until [ "$(kubectl get restore k8s-backups-users-resources-restore-$RESTORE_SUFFIX -n velero -o jsonpath='{.status.phase}' 2>/dev/null)" = "Completed" ]; do
+  echo "Still waiting..."; sleep 5;
+done
 ```
 
-#### Restore Users and Grants
+### Restore on Cluster installation
 
-In the backup phase, we took the backup of the user and grants separately.
-The last step of the RonDB restore process is to re-create all the users and grants both for Hopsworks services as well as for the online feature store users.
-This can be achieved by running the following command on one node of the head node group:
+To restore a cluster during installation, configure the backup ID in the values YAML file:
 
-```sh
-/srv/hops/mysql-cluster/ndb/scripts/mysql-client.sh source users.sql
+```yaml
+global:
+  _hopsworks:
+    backups:
+      enabled: true
+      schedule: "@weekly"
+    restoreFromBackup:
+      backupId: "254811200"
```

Comment:
I suppose there is a way to find the backup ID by getting the Velero backups? Can you show an example before this?

Reply:
yeah, done, added the info at the end of velero backup restore. Although i would imaging this to be better planned as well before termintaing the cluster so it should be visible from the UI